tetrateci/tsb-util/templates/k8s-objects/bridged/servicerouteeditor.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ servicerouteSAName }}
+  namespace: tsb
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ podName }}
+  namespace: tsb
+spec:
+  containers:
+  - image: amd64/ubuntu:focal
+    name: tctl-container
+    command: [ "/bin/bash", "-c", "--" ]
+    args:
+    - |
+      apt update && apt install curl -y
+      curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+      IP=$(kubectl get svc envoy -n tsb -o "jsonpath={.status.loadBalancer.ingress[0].ip}")
+      echo "IP = $IP"
+      mkdir -p ~/.tctl/bin
+      curl -Lo ~/.tctl/bin/tctl https://binaries.dl.tetrate.io/public/raw/versions/linux-amd64-1.2.0/tctl
+      chmod +x ~/.tctl/bin/tctl
+      export PATH=$PATH:~/.tctl/bin
+      tctl config clusters set demo --bridge-address $IP:8443
+      tctl config users set admin --username admin --password "{{ password }}"
+      tctl config profiles set demo --username admin --cluster demo
+      tctl config profiles set-current demo
+      tctl login --org {{ orgName }} --password "{{ password }}" --username admin --tenant {{ tenant }}
```

Comment:
How are you getting the tenant name?

Reply:
tenant is configured while generating the configs, since we have one serviceroute editor for one bookinfo installation, I just reuse the tenant for that particular bookinfo installation